### ISSUE TITLE
Whitelist flags for generate command

### ISSUE THREAD

**Issue by K1DV5** [go-jet/jet#216]
**Is your feature request related to a problem? Please describe.**
I wanted to generate models for a few tables in a big database. I understand that there is the `-ignore-tables` flag, but it is too much work to list the 45 tables to generate models for 5 tables in a 50 table database. And if a new table is added to the database and I don't want to include it in the models, I would have to remember to include it as the 46th table to ignore.

**Describe the solution you'd like**
I think it'd be great if there was something like `-allow-tables` flag which would take precedence over `-ignore-tables` when given. And same applies about enums and views.

**Comment by go-jet** [comment]
Yeah, it makes sense. At the moment this behavior can be also achieved using generator customization([model example](https://github.com/go-jet/jet/blob/f2e4b8551c48b97d0cd2d3deff47dc1b2aa2f04e/tests/postgres/generator_template_test.go#L173), [sql builder example](https://github.com/go-jet/jet/blob/f2e4b8551c48b97d0cd2d3deff47dc1b2aa2f04e/tests/postgres/generator_template_test.go#L201)).
